src/test/java/org/gridsuite/filter/server/FilterEntityControllerTest.java

```diff
@@ -830,8 +830,6 @@ public void testIdentifierListFilter() throws Exception {
 
         // Create identifier list filter for hvdc
         UUID hvdcFilterId = UUID.randomUUID();
-        IdentifierListFilterEquipmentAttributes hvdc1 = new IdentifierListFilterEquipmentAttributes("threeWT1", null);
-        IdentifierListFilterEquipmentAttributes hvdc2 = new IdentifierListFilterEquipmentAttributes("threeWT2", null);
```

Comment:
Oh I guess that's why filterId3 is empty now, why did you delete these ?

Reply:
Not used